web-console/4.0-designs/create-from-YAML/create-from-YAML.md

```diff
@@ -0,0 +1,35 @@
+# Create from YAML
+
+A Create from YAML button will be added to the project selector bar, making it accessible from every page within the console.
+
+
+**Create from YAML button**
+![button](img/YAML-Button.png)
+
+* The Create from YAML button will be in the project selector bar
```

Comment:
Should we change the label to "Create YAML" or just "Create" so it's not quite as eye-catching with the + icon lining up directly with the help icon above it? Just an idea. 

I feel like it may be fine not having this action available on the projects and namespaces page. It would be available on every other page I believe. We could implement as is for 4.0 and get some feedback from customers. If it is used heavily, then we can always add it back into the catalog somewhere.